pkg/addons/adminconsole/adminconsole.go

```diff
@@ -39,7 +39,7 @@ var (
 )
 
 // protectedFields are helm values that are not overwritten when upgrading the addon.
-var protectedFields = []string{"automation", "embeddedClusterID"}
+var protectedFields = []string{"automation", "embeddedClusterID", "embeddedClusterVersion"}
```

Comment:
this should not be protected, doing so will prevent it from changing when we update embedded-cluster

Reply:
whoops. good catch.